enterprise/reporting/label_set_test.go

```diff
@@ -65,12 +93,12 @@ func TestGenerateHash(t *testing.T) {
 		assert.Equal(t, hash1, hash2)
 	})
 
-	t.Run("different hash for different LabelSet", func(t *testing.T) {
-		inputMetric1 := createMetricObject("some-event-name-1")
+	t.Run("different hash for different LabelSet for errors", func(t *testing.T) {
+		inputMetric1 := createMetricObject("some-event-name-1", true)
 		bucket := int64(28889820)
 		labelSet1 := NewLabelSet(inputMetric1, bucket)
 
-		inputMetric2 := createMetricObject("some-event-name-2")
+		inputMetric2 := createMetricObject("some-event-name-2", true)
```

Comment:
minor: ideally we should test with same event name and different errorCode / errorMessage

Reply:
Updated